### PR TITLE
adding fleet service

### PR DESCRIPTION
**Pull request by PierreZ**
I'm using a CoreOS instance. This fleet file allows docker-gc to be run on every instance of my cluster.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,3 +23,4 @@ week-end. You can override with `--maxAge [duration]` using Golang duration synt
 For your convenience, docker-gc is also available as a docker image. You can run it as `docker run -d -v /var/run/docker.sock:/var/run/docker.sock ndeloof/docker-gc`
 You need to bind mount `/var/run/docker.sock` from docker host so the docker-gc process can attach to the daemon to listen for event and inspect/remove images.
 
+You can also use `docker-gc.service` to deploy Docker-gc everywhere using [Fleet](https://coreos.com/fleet/docs/latest/) on a CoreOS cluster. Just run `fleetctl start docker-gc.service`.
```
Comment (tbroyer) [inline README.md:26]: I was about to ask for a SystemD service unit to easily "install" it on my laptop, so it's not only about Fleet.

File: docker-gc.service [hunk 2]
```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description=Docker-gc Service
+After=docker.service
+Requires=docker.service
+[Service]
+Restart=always
+ExecStartPre=-/usr/bin/docker kill docker-gc
+ExecStartPre=-/usr/bin/docker rm docker-gc
+ExecStartPre=/usr/bin/docker pull ndeloof/docker-gc:latest
```
Comment (tbroyer) [inline docker-gc.service:9]: I'm curious: will docker-gc be able to correctly collect previous versions of itself when updated? (I admin this is not really related to this PR, but reading this line rang that bell)

Comment (PierreZ) [inline docker-gc.service:9]: I'm cheating in fact. CoreOS is all about reboot for self-upgrading. So it's pulling at each reboot the latest image ;-)

**Comment by ndeloof** [comment]
@tbroyer docker-gc only consider running images and the one it can collect infos about with docker events. It could be enhanced for this purpose, see https://github.com/ndeloof/docker-gc/issues/3
